### ISSUE TITLE
After 1.1.2 style is broken

### ISSUE THREAD

**Issue by fabiot21** [cuire/svelte-grid-extended#55]
#52 broke the style of an internal app. the grid no longer consider the total height and width of all items as a container 🥹 @cuire   Is it possible to rollback? or any other way to add those styles on client side?

I think the deletion of this is what causes the style bug now.

```js
$: if ($gridSettings.itemSize) containerWidth = _cols * ($gridSettings.itemSize.width + gap + 1);
$: if ($gridSettings.itemSize) containerHeight = _rows * ($gridSettings.itemSize.height + gap + 1);
```

REPL working on 1.1.1 https://svelte.dev/repl/09ea1a4b3f964f209d22c9ebe6d3b82c?version=4.2.1 (not working correctly with 1.1.2 / 1.1.3)

**Comment by cuire** [comment]
I know how to fix that, wait for PR

**Comment by cuire** [comment]
Fixed in 1.1.4. Thanks for reporting the issue 💖
